src/libraries/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/Machine.cs

```diff
@@ -129,5 +129,10 @@ public enum Machine : ushort
         /// ARM64
         /// </summary>
         Arm64 = 0xAA64,
+
+        /// <summary>
+        /// LOONGARCH64
+        /// </summary>
+        LoongArch64 = 0x6264,
```

Comment:
It may make sense to also add `LoongArch32` to this enum since it is part of the PE file spec now. This enum should mirror PE file spec, independent on whether .NET runs on the architecture or not.

Reply:
Thanks, I will update it.